flink-end-to-end-tests/flink-quickstart-test/pom.xml

```diff
@@ -86,6 +102,21 @@ under the License.
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-enforcer-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>dependency-convergence</id>
+						<goals>
+							<goal>enforce</goal>
+						</goals>
+						<configuration>
+							<skip>true</skip>
```

Comment:
this was a good hint that what you were trying to do was problematic

Reply:
@zentol good point. this is not necessary anymore